doc/sphinx/Pacemaker_Development/helpers.rst

```diff
@@ -468,6 +468,117 @@ At this point, you need to determine whether your test case is incorrect or
 whether the code being tested is incorrect.  Fix whichever is wrong and continue.
 
 
+Fuzz Testing
+############
+
+Pacemaker is integrated with the
+`OSS-Fuzz <https://github.com/google/oss-fuzz>`_ project. OSS-Fuzz calls
+selected Pacemaker APIs with random argument values to catch edge cases that
+might be missed by other forms of testing.
+
+The OSS-Fuzz project has a contact address for Pacemaker in
+projects/pacemaker/project.yaml that will receive bug reports. The address must
+have been used to commit to Pacemaker, and should be tied to a Google account.
+
+Open reports that aren't security-related can be seen at `OSS-Fuzz testcases
+<https://oss-fuzz.com/testcases?project=pacemaker&open=yes>`_.
+
+
+Fuzzers
+_______
+
+Each fuzz-tested library has a fuzzers subdirectory (for example,
+``lib/common/fuzzers``). That directory has a file for each fuzzed source file,
+named the same except ending in ``_fuzzer.c`` (for example,
+``lib/common/fuzzers/strings_fuzzer.c`` has fuzzing for
+``lib/common/strings.c``). Those files are not built or distributed as part of
+Pacemaker but are used by OSS-Fuzz (see ``projects/pacemaker/build.sh`` in the
+OSS-Fuzz repository).
+
+By default, fuzzing uses `libFuzzer <https://llvm.org/docs/LibFuzzer.html>`_.
+Only Pacemaker APIs that accept any input and do not exit can be fuzzed.
+Ideally, fuzzed functions will not modify global state or vary code paths by
+anything other than the fuzzed input.
```

Comment:
One simple example - admittedly hard to avoid but a good example for getting the idea - is usage
of the current time in the code.

Reply:
added a couple examples, thanks

Reply:
Guess that be enough to give an idea in which direction to think.